### DISCUSSION TITLE
Start arcadedb with tcp/ip and use orientdb driver

### DISCUSSION THREAD

**Discussion by centoria** [ArcadeData/arcadedb#1035] [Q&A]
Hi is it possible to start arcadb with tcp/ip connection in 2424 port like orientdb and use orientdb driver ??

I tried this, i have a project running with orientdb and i want to migrate to arcadedb without changing my code base is it possible ?

thks

**Comment by gramian** [comment]
Hi,
you can adapt the port with the [`server.httpIncomingPort`](https://docs.arcadedb.com/#_server-2) setting. However, I assume ArcadeDB is likely not a drop-in replacement for OrientDB, for example OrientDB's SQL `CLASS` is in ArcadeDB's SQL `TYPE`. I am also not confident the OrientDB driver will work on ArcadeDB, but @lvca can answer that more definitely. Given ArcadeDB's relation to OrientDB the migration may only need minor adaptions, yet it is hard to tell without more details.

**Comment by lvca** [comment]
ArcadeDB doesn't have a binary TCP/IP driver like OrientDB. You can use the `RemoteDatabase` interface with some nice abstraction with the remote calls. I know, it's not a drop-in replacement, but at least you can use a Java interface.

**Reply by centoria** [reply]
Thanks so i have to adapt my code base.
